CHANGELOG.md

```diff
@@ -9,13 +9,15 @@ Change log
 o'clock uses [PVP Versioning][1].
 The change log is available [on GitHub][2].
 
-Unreleased
+1.4.0
```

Comment:
We should update this as well:

> I updated the version number in `o-clock.cabal`.

Reply:
Oh, yep, actually I wanted to mark it as a draft while I'm messing with CI, but you were quite fast :D
I'll do it